_posts/2017-11-16-negative-energy-flood.markdown

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title:  "Negative Energy Flood"
+date:   2017-11-16
+source: Xanathar.163
+tags: [warlock, wizard, level5, necromancy]
+---
+
+**5th-level necromancy**
+
+**Casting Time**: 1 action
+
+**Range**: 60 feet
+
+**Components**: V, M (a broken bone and a square of black silk)
+
+**Duration**: Instantaneous 
+
+You send ribbons of negative energy at one creature you can see within range. Unless the target is undead. it must make a Constitution saving throw, taking 5d12 necrotic damage on a failed save, or half as much damage on a successful one. A target killed by this damage rises up as a zombie at the start of your next turn. The zombie pursues whatever creature it can see that is closest to it. Statistics for the zombie are in the Monster Manual. 
```

Comment:
You need "Unless the target is undead, it" not "Unless the target is undead. it"